config/locales/en.yml

```diff
@@ -1,3 +1,4 @@
+---
```

Comment:
This is seems quite a large automatic change - could you add a note in the commit for this that explains why the strings don't need to be quoted?

Reply:
Sure, I put what I knew already into the commit message but i might need to dig a little more into [psych](https://github.com/ruby/psych) to work out the real reasoning.

The quick version is that if you use the ruby yaml lib to import the CSV file currently in master, then immediately export it back to yaml and write that to a file. This is what you get.

There are three tests in psych that I can see that might be of interest:
- [test_no_doublequotes_with_special_characters](https://github.com/ruby/psych/blob/master/test/psych/test_string.rb#L24)
-[test_no_quotes_when_start_with_non_ascii_character](https://github.com/ruby/psych/blob/master/test/psych/test_string.rb#L28)
-[test_doublequotes_when_there_is_a_single](https://github.com/ruby/psych/blob/master/test/psych/test_string.rb#L33)

The TLDR though, is that we want to be able to import future changes from the google sheet and get a very simple git diff, we don't want it flagging output that has different quote formats or line breaks, otherwise it's going to get messy for any dev doing a commit.

Reply:
I'll add that to the commit message

Reply:
Looking over the file the output rules seem to be:
- If you have multiple words on a single line, no quotes
- If you have a single word starting with an ASCII 2 character then single quote it
- If you have multiple words containing a special character single quote it
- if it's a URL, don't quote it

Reply:
I pulled up the results pages and started manually taking screenshots of these to show they rendered ok.
Stopped about a quater of the way down. After seeing enough I'm confident these render OK!